src/PSRule.Rules.Azure/Configuration/ConfigurationOption.cs

```diff
@@ -63,7 +66,8 @@ public bool Equals(ConfigurationOption other)
                 Tenant == other.Tenant &&
                 ManagementGroup == other.ManagementGroup &&
                 ParameterDefaults == other.ParameterDefaults &&
-                Deployment == other.Deployment;
+                Deployment == other.Deployment &&
+                PolicyRulePrefix.Equals(other.PolicyRulePrefix, StringComparison.OrdinalIgnoreCase);
```

Comment:
This is for object equity `==` is fine.

```suggestion
                PolicyRulePrefix == other.PolicyRulePrefix;
```